pwiz_tools/Skyline/Properties/Resources.Designer.cs

```diff
@@ -5593,7 +5593,13 @@ public class Resources {
                 return ResourceManager.GetString("CommandLine_ImportAnnotations_Error__Failed_while_reading_annotations_", resourceCulture);
             }
         }
-        
+
+        public static string CommandLine_ImportPeakBoundaries_Error__Failed_while_importing_boundaries_ {
+            get {

```

Comment:
Redo this by adding a string literal and then pressing F6 to have Resharper move it to Properties\Resources.resx which will also create this property.